packages/autocomplete-core/src/__tests__/getInputProps.test.ts

```diff
@@ -1893,81 +1918,6 @@ describe('getInputProps', () => {
     });
   });
 
-  describe('onBlur', () => {
```

Comment:
The test cases from this sub-suite are now split across multiple tests in the `onKeyDown` sub-suite and  the `getEnvironmentProps` suite.